lib/ja_serializer/param_parser.ex

```diff
@@ -37,11 +37,16 @@ defmodule JaSerializer.ParamParser.Utils do
   def format_key(key) do
     case Application.get_env(:ja_serializer, :key_format, :dasherized) do
       :dasherized -> dash_to_underscore(key)
+      :camelcased -> uppercase_to_underscore(key)
       :underscored -> key
       {:custom, module, _, fun} -> apply(module, fun, [key])
       _ -> key
     end
   end
 
   defp dash_to_underscore(key), do: String.replace(key, "-", "_")
+
+  defp uppercase_to_underscore(key) do
+    Macro.underscore(key)
```

Comment:
This is another one where the docs don't recommend using it for general underscore purposes. I would feel more comfortable if we did not use here for that reason.